src/lib/error.ts

```diff
@@ -86,3 +87,14 @@ export class InclusionAlreadyInProgressError extends BaseError {
 export class InvalidParamsPassedToCommandError extends BaseError {
   errorCode = ErrorCode.invalidParamsPassedToCommand;
 }
+
+export class NoLongerSupportedError extends BaseError {
+  errorCode = ErrorCode.noLongerSupported;
+
+  constructor(public customMessage: string) {
+    super(
+      customMessage +
+        " If you are using an application that integrates with Z-Wave JS and you receive this error, you may need to update the application.",
```

Comment:
is the `customMessage` property actually used anywhere? If not, I wouldn't create another property for it:
```suggestion
  constructor(message: string) {
    super(
      message +
        " If you are using an application that integrates with Z-Wave JS and you receive this error, you may need to update the application.",
```
After this, the `error.message` property will be the full, concatenated string.